aptos-move/mvhashmap/src/versioned_delayed_fields.rs

```diff
@@ -182,6 +174,12 @@ impl<K: Copy + Clone + Debug + Eq> VersionedValue<K> {
         }
     }
 
+    fn insert_final_value(&mut self, txn_idx: TxnIndex, value: DelayedFieldValue) {
+        if let None = self.versioned_map.insert(txn_idx, CachePadded::new(VersionEntry::Value(value, None))) {
```

Comment:
Let's still assert (I guess we will change to CodeError) more than not None, it should be a delta (or maybe a value with the same value). Or maybe that it's not an estimate.